docs/src/main/paradox/index.md

```diff
@@ -0,0 +1,18 @@
+# Dart
```

Comment:
Let's have the main title as `Cloudstate Dart`, which is the pattern we're using for other modules like Go and Python.

Reply:
Thanks @pvlugter